doc/spec/inference.rst

```diff
@@ -153,7 +154,7 @@ estimating the uncertainty of the honest causal forests underlying the OrthoFore
                                          min_leaf_size=3,
                                          model_T=WeightedLasso(alpha=0.01),
                                          model_Y=WeightedLasso(alpha=0.01))
-    est.fit(y, t, X, W, inference='blb')
+    est.fit(y, t, X, W)
```

Comment:
@vsyrgkanis This change is causing a doctest failure because ContinuousTreatmentOrthoForest doesn't use 'auto' as a default (as opposed to DiscreteTreatmentOrthoForest, which does).  Is there a reason that it doesn't, or is that just an oversight?

Reply:
Oversight! Please change

Reply:
I've addressed this, although the fix has highlighted how weird our `wrap_fit` logic is; I think we should probably completely revamp it as part of addressing the second-stage-only bootstrap changes.